packages/dcos-integration-test/extra/util/delete_ec2_volume.py

```diff
@@ -41,29 +45,52 @@ def _remove_env_vars(*env_vars):
 
 @retrying.retry(
     wait_exponential_multiplier=1000,
-    wait_exponential_max=300000,
-    stop_max_delay=1800000,
+    wait_exponential_max=300 * 000,
```

Comment:
isn't this value simply 0? same for the following

looks like typos to me

Reply:
Ha! Great catch! Thanks James! :man_facepalming: